kong/plugins/log-serializers/basic.lua

```diff
@@ -26,7 +26,10 @@ function _M.serialize(ngx)
       querystring = req.get_uri_args(), -- parameters, as a table
       method = req.get_method(), -- http method
       headers = req.get_headers(),
-      size = var.request_length
+      size = var.request_length,
+      tls = {
+        version = ctx.KONG_TLS_VERSION
```

Comment:
For some reason this value is always `nil`, even when properly being set in `certificate.lua` with:

```lua
ngx.ctx.KONG_TLS_VERSION = ver
```

Reply:
It reminds me of this:
https://github.com/Kong/kong/issues/4379

Reply:
Perhaps contact `upstream` and ask if `ngx.ctx` should work with `ssl_certicate`:
https://github.com/openresty/lua-nginx-module#ngxctx

On my testing `rewrite` resets `ngx.ctx`.